static/styles/subscriptions.css

```diff
@@ -832,10 +803,6 @@ h4.stream_setting_subsection_title {
         }
     }
 
-    .inner-box {
-        margin: 20px;
-    }
-
```

Comment:
Removed this `margin` from `.inner-box` class and added it instead in `.stream_section` nested CSS class with the `padding-left: 2px` to make the visibility of focus outline in checkboxes a bit clear.